examples/common/pigweed/chip_common_proto/chip_common.proto

```diff
@@ -0,0 +1,5 @@
+syntax = "proto3";
+
+package chip.rpc;
```

Comment:
nitpicky: I would try to avoid names like common/helpers/support/util. Also chip_common_proto/chip_common stutters (even more so as we are in examples/common).

`proto/empty.proto` ?

Reply:
Done